plugins/source/oracledb/client/sync.go

```diff
@@ -27,22 +29,23 @@ func (*Client) createResultsArray(table *schema.Table) []any {
 	results := make([]any, 0, len(table.Columns))
 	for _, col := range table.Columns {
 		switch col.Type {
-		case schema.TypeUUID, schema.TypeByteArray:
+		case types.ExtensionTypes.UUID, arrow.BinaryTypes.Binary, arrow.BinaryTypes.LargeBinary:
```

Comment:
fixed size binary?

Reply:
Cleaned this up in https://github.com/cloudquery/cloudquery/pull/11115/commits/18f83f179999630a6272f36892a9317d50651bcd as we only need to support the arrows types we use based on the DB schema